### DISCUSSION TITLE
Dogecoin Core Documentation and Translation

### DISCUSSION THREAD

**Discussion by qlpqlp** [dogecoin/dogecoin#3450] [General]
![image](https://github.com/dogecoin/dogecoin/assets/5968076/bae500a6-bc49-4023-98a0-d2ccbbffc25c)

Hello, Like talked here https://github.com/dogecoin/dogecoin/discussions/3436 I have step forward and organised all **CORE** Documentations to be easily translated to any language.

I also have created the domains [dogecoin-core.com](https://dogecoin-core.com) and [dogecoincore.com](https://dogecoincore.com) to easily access to decentralize even more **Dogecoin Core**.

I'm open to any suggestions or ideas, the GITHUB repo is here: [https://github.com/qlpqlp/dogecoin-core-docs](https://github.com/qlpqlp/dogecoin-core-docs)

I will need any help on translate, its easy, duplicate the folder **src/en/** and change the name **en** to your language and translate all .md files and PR to it :D

**Comment by patricklodder** [comment]
Could you please respect the license? You're now re-assigning copyright to yourself without respecting the works of others you took from here.

Otherwise, good job.

**Reply by qlpqlp** [reply]
Much thanks @patricklodder 
Sorry I did not did on purpose the licencing, it was auto generated by GIT, I'm still not very good at it, but learning :)

I think its now good: https://github.com/qlpqlp/dogecoin-core-docs/blob/main/LICENSE

**Reply by patricklodder** [reply]
Yup, thanks.

Minor thing: I think you took the (c) lines from `master` but the sources from `1.14.7-dev` (otherwise there wouldn't be `vn`, right?) So the correct end year is 2024.

**Reply by qlpqlp** [reply]
Yes lol
Fixed :P

**Comment by 0foxes** [comment]
It seems to me that having domains [dogecoin-core.com](https://dogecoin-core.com/) and [dogecoincore.com](https://dogecoincore.com/) may be dangerous as it creates confusion for users, unless accompanied by a disclaimer that it is not an "official" site.

**Reply by qlpqlp** [reply]
Why do I need to add the disclaimer if its the Dogecoin Core documentation?
I don't see anything dangerous.

**Reply by daank-c** [reply]
best practice is to cite primary sources. this way people can verify that the information is what it claims to be.

Many example of this, but here's a relevant one from https://en.bitcoin.it/wiki/BIP_0300
![IMG_0683](https://github.com/dogecoin/dogecoin/assets/102710924/576a9d0a-b48c-4343-be4a-f26a702c93e6)

**Reply by qlpqlp** [reply]
how do you propose to do that? Do you want to help me on that matter? GitHub is open and this Documentation is for every shibe that want to learn how Dogecoin Core works 🙂

**Reply by daank-c** [reply]
I'm not sure what everyone else thinks, but in my opinion, it would be awesome if there was something like this:


![Screenshot 2024-02-29 3 03 59 AM](https://github.com/dogecoin/dogecoin/assets/102710924/35708d17-4f85-429f-a433-87afd9e143d7)


How to implement:

index.html  (Line 184)


```
+      <div class="disclaimer-box">
+           <p>This page was last updated on February 28th, 2024.</p>
+	      <p>The latest version of this documentation can be found <a href="https://github.com/dogecoin/dogecoin">here</a>.</p>
+      </div>
```

css/general.css (at the end)

```
+
+       .disclaimer-box {
+            background-color: #f2f2f2;
+            padding: 10px;
+            margin: 10px 0;
+            border-radius: 5px;
+            font-size: 14px;
+            text-align: center
+     }
+
+     .disclaimer-box p {
+           margin: 0;
+ 	     color: #333;
+     }
```

**Reply by qlpqlp** [reply]
If the documentation was pushed to CORE in a new repo, nothing of this is needed and the Documentation would be were it belongs 🙂

**Reply by georgeartem** [reply]
@daank-c I've implemented your suggestion on the dogedealz home page. It's a good one.

**Comment by georgeartem** [comment]
The attribution license in this repository has been placed in the public domain under an open source license by those who have made contributions in the last decade.The update to the copyright in the latest release expresses that pretty explicitly.Exactly what mechanism you implement to portray that is largely up to you, but I think the concerns expressed in this thread speak to some of the recent history surrounding the positions taken by your employer on these types of legal issues.I don’t know if it’s necessary, but what might be helpful for folks here to understand is if you are acting on your own behalf as part of making your own individual contributions to core or if you are acting as an employee/agent of your employer that may or may not later intend to lay claims to the documentation (already distributed to the public domain) that you now host separately from this repository.Hope that is helpful 🙏Sent from my iPhoneOn Feb 28, 2024, at 12:09 AM, Paulo Vidal ***@***.***> wrote:﻿
how do you propose to do that? Do you want to help me on that matter? GitHub is open and this Documentation is for every shibe that want to learn how Dogecoin Core works 🙂

**Reply by qlpqlp** [reply]
Mix feelings:

1°- My contribution Its open source
2°- Dogecoin Foundation projects are Opensource
3°- My CORE documentation improvements and contribution is on my personal GITHUB account not in Foundation, probably will push to dogeorg account if not pushed to dogecoin core as it should.
4°- Dogecoin supposedly its decentralized so the more the merrier
5°- I did not asked any money for what I did
6° My latest CORE contribution as a CORE dev in 2022 I was paid from the CORE tip jar 40,000 Doge that once again gave everything to everyone in the community 
7°- even not believing personal on CORE as the best software to communicate with Dogecoin BlockChain, I still know its used by many and its an entry point for anyone that wants to initiate development, thats why I have improved the Documentation that was really bad organized since 2013 and only now its a little better.

**Comment by georgeartem** [comment]
My thoughts:

1. Awesome
2. This is more akin to "gated source" in comparison to the CORE repo.
3. Ok
4. Ok
5. Ok. What's your compensation structure like? Are you salaried? Hourly?
Are you receiving a performance based bonus? Have representations been made
to you concerning directorship, board positions etc?
6. Cool that CORE has historically offered bounties for work they believed
was worthwhile.
7. Where are you going with that?

**Reply by qlpqlp** [reply]
> My thoughts:
> 
> 1. Awesome
> 2. This is more akin to "gated source" in comparison to the CORE repo.
> 3. Ok
> 4. Ok
> 5. Ok. What's your compensation structure like? Are you salaried? Hourly?
> Are you receiving a performance based bonus? Have representations been made
> to you concerning directorship, board positions etc?
> 6. Cool that CORE has historically offered bounties for work they believed
> was worthwhile.
> 7. Where are you going with that?
> 
> 
> On Thu, Feb 29, 2024 at 1:13 AM Paulo Vidal ***@***.***>
> wrote:
> 
> > Mix feelings:
> >
> > 1°- My contribution Its open source
> > 2°- Dogecoin Foundation projects are Opensource
> > 3°- My CORE documentation improvements and contribution is on my personal
> > GITHUB account not in Foundation, probably will push to dogeorg account if
> > not pushed to dogecoin core as it should.
> > 4°- Dogecoin supposedly its decentralized so the more the merrier
> > 5°- I did not asked any money for what I did
> > 6° My latest CORE contribution as a CORE dev in 2022 I was paid from the
> > CORE tip jar 40,000 Doge that once again gave everything to everyone in the
> > community
> > 7°- even not believing personal on CORE as the best software to
> > communicate with Dogecoin BlockChain, I still know its used by many and its
> > an entry point for anyone that wants to initiate development, thats why I
> > have improved the Documentation that was really bad organized since 2013
> > and only now its a little better.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/dogecoin/dogecoin/discussions/3450#discussioncomment-8628159>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/ACZSY4Y4M6VESQSKI2Y5AUTYV3YMVAVCNFSM6AAAAABD4M6ELOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DMMRYGE2TS>
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
> 

5. The same I can ask, do you work to get money to put food on the table? What I did was because I wanted to do to show that anyone can do things if they really want to help without any second intentions. I'm mostly frustrated because people thinks that Dogecoin is owned or controlled and its not, anyone can do anything.

6. Are not bounties, are simply tips that someone sended to ve ised on all CORE devs.

7. Anyone can build anything, no one have to wait, no one have to expect anyone to build something for you

**Comment by georgeartem** [comment]
Responses:5. I agree the accessibility of these projects is hard without financial backing, but you’re deflecting the question.6. Huh? I need more context.7. See response to 5Sent from my iPhoneOn Mar 16, 2024, at 1:34 AM, Paulo Vidal ***@***.***> wrote:﻿

My thoughts:

Awesome
This is more akin to "gated source" in comparison to the CORE repo.
Ok
Ok
Ok. What's your compensation structure like? Are you salaried? Hourly?
Are you receiving a performance based bonus? Have representations been made
to you concerning directorship, board positions etc?
Cool that CORE has historically offered bounties for work they believed
was worthwhile.
Where are you going with that?

On Thu, Feb 29, 2024 at 1:13 AM Paulo Vidal @.***>
wrote:

Mix feelings:
1°- My contribution Its open source
2°- Dogecoin Foundation projects are Opensource
3°- My CORE documentation improvements and contribution is on my personal
GITHUB account not in Foundation, probably will push to dogeorg account if
not pushed to dogecoin core as it should.
4°- Dogecoin supposedly its decentralized so the more the merrier
5°- I did not asked any money for what I did
6° My latest CORE contribution as a CORE dev in 2022 I was paid from the
CORE tip jar 40,000 Doge that once again gave everything to everyone in the
community
7°- even not believing personal on CORE as the best software to
communicate with Dogecoin BlockChain, I still know its used by many and its
an entry point for anyone that wants to initiate development, thats why I
have improved the Documentation that was really bad organized since 2013
and only now its a little better.
—
Reply to this email directly, view it on GitHub
#3450 (reply in thread),
or unsubscribe
https://github.com/notifications/unsubscribe-auth/ACZSY4Y4M6VESQSKI2Y5AUTYV3YMVAVCNFSM6AAAAABD4M6ELOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DMMRYGE2TS
.
You are receiving this because you commented.Message ID:
@.***>




The same I can ask, do you work to get money to put food on the table? What I did was because I wanted to do to show that anyone can do things if they really want to help without any second intentions. I'm mostly frustrated because people thinks that Dogecoin is owned or controlled and its not, anyone can do anything.


Are not bounties, are simply tips that someone sended to ve ised on all CORE devs.


Anyone can build anything, no one have to wait, no one have to expect anyone to build something for you

**Reply by qlpqlp** [reply]
5- Not deflecting, you are hoping me to reply that I'm being paid to do this, I'm not

6- Donations / tips, are not bounties 

7-👍

**Comment by georgeartem** [comment]
Right. Not deflecting anymore. What you’re doing is called stonewalling. Sent from my iPhoneOn Mar 16, 2024, at 8:05 AM, Paulo Vidal ***@***.***> wrote:﻿
5- Not deflecting, you are hoping me to reply that I'm being paid to do this, I'm not
6- Donations / tips, are not bounties
7-👍

**Reply by qlpqlp** [reply]
![Screenshot_20240316_154638_Google.jpg](https://github.com/dogecoin/dogecoin/assets/5968076/2dab6e67-1136-4b67-8952-d163a6521e7b)

🤣 How I'm not answering? I said declared that I'm not being paid for doing what I did on the Documentation, actually I'm spending because I paid for the Domain, I'm paying for the server and using my own time on doing things to show that can be better ¯\_(ツ)_/¯

I call that questions you asked to be unnecessary questions because are just lame and offensive in my POV

**Comment by georgeartem** [comment]
You’re right. Carry on 👍Sent from my iPhoneOn Mar 16, 2024, at 8:50 AM, Paulo Vidal ***@***.***> wrote:﻿
Screenshot_20240316_154638_Google.jpg (view on web)
🤣 How I'm not answering? I said declared that I'm not being paid for doing what I did on the Documentation, actually I'm spending because I paid for the Domain, I'm paying for the server and using my own time on doing things to show that can be better ¯_(ツ)_/¯
I call that questions you asked to be unnecessary questions because are just lame and offensive in my POV
